client/my-sites/themes/theme-showcase.jsx

```diff
@@ -58,6 +60,98 @@ const optionShape = PropTypes.shape( {
 	action: PropTypes.func,
 } );
 
+const bannerShape = {
+	siteId: PropTypes.number,
+};
+
+const bannerClickHandler = ( themeId, tracksEvent, { siteId, getBannerUrl, getCurrentTheme } ) => {
+	const url = getBannerUrl( themeId, siteId );
+	const tracksData = {
+		site_id: siteId,
+		current_theme: getCurrentTheme( siteId ),
+		timestamp: new Date().toGMTString(),
```

Comment:
Do we need to do this? I was kind of assuming tracks would take care of adding something like a timestamp for us automagically. I might need to verify that assumption...

Reply:
![image](https://user-images.githubusercontent.com/96308/35880832-dbd852b0-0b76-11e8-9020-3dea1b5ffb55.png)

Looks like it does :slightly_smiling_face: